baseplate/thrift/BaseplateService.py

```diff
@@ -1,5 +1,5 @@
 #
-# Autogenerated by Thrift Compiler (0.12.1)
+# Autogenerated by Thrift Compiler (0.13.0)
```

Comment:
@spladug is this ok? 😬

Reply:
seems fine. only change is the `_on_message_begin()` thing which looks harmless